app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,48 @@
+<main>
+  <div>
+    <header class="search-wrapper">
+      <%= link_to root_path do %>
+      <h1 class="margin-zero" id="large-font">recipe h<em>API</em>
+        <small>( ˘▽˘)っ♨</small>
```

Comment:
You include this search bar on every page! You should DRY this up by putting it in `layouts/application.html.erb`